code/modules/projectiles/guns/chaos_bolt.dm

```diff
@@ -0,0 +1,340 @@
+#define CHAOS_STAFF_DAMAGE 30 //Damaging effects will use this number, multiplied or divided depending on effect
+#define CHAOS_STAFF_LETHAL_CHANCE 5 //These should add up to 100
+#define CHAOS_STAFF_NEGATIVE_CHANCE 45
+#define CHAOS_STAFF_MISC_CHANCE 30
+#define CHAOS_STAFF_GIFT_CHANCE 15
+#define CHAOS_STAFF_GREAT_GIFT_CHANCE 5
```

Comment:
No need to undefine these later?

Reply:
Replacing these defines with vars since it doesnt seem these are usually used in this way